lib/lightning_web/live/first_setup_live/superuser.ex

```diff
@@ -66,7 +66,7 @@ defmodule LightningWeb.FirstSetupLive.Superuser do
     if Lightning.Accounts.has_one_superuser?() do
       socket
       |> put_flash(:warn, "Superuser account already exists.")
-      |> push_redirect(to: Routes.dashboard_index_path(socket, :index))
+      |> push_redirect(to: Routes.dashboard_index_path(socket, :show))
```

Comment:
I think we should be redirecting to the index by default, and _if_ a user has a project, then redirect again to the show page.

I'm getting the sense that the index page's template is a fallback.